README.md

```diff
@@ -2,13 +2,13 @@
 
 [![GitHub Actions CI Status](https://github.com/nf-core/epitopeprediction/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/epitopeprediction/actions?query=workflow%3A%22nf-core+CI%22)
 [![GitHub Actions Linting Status](https://github.com/nf-core/epitopeprediction/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/epitopeprediction/actions?query=workflow%3A%22nf-core+linting%22)
-[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/epitopeprediction/results)
-[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.3564666-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.3564666)
+[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/epitopeprediction/results)[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.3564666-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.3564666)
 
-[![Nextflow](https://img.shields.io/badge/nextflow%20DSL2-%E2%89%A521.10.3-23aa62.svg?labelColor=000000)](https://www.nextflow.io/)
+[![Nextflow](https://img.shields.io/badge/nextflow%20DSL2-%E2%89%A521.10.3-23aa62.svg)](https://www.nextflow.io/)
```

Comment:
Some of your labels are now black, some are grey. I'd propose make them all black, such as:
```suggestion
[![Nextflow](https://img.shields.io/badge/nextflow%20DSL2-%E2%89%A521.10.3-23aa62.svg?labelColor=000000)](https://www.nextflow.io/)
```

Reply:
I think this might be wrong in the template

Reply:
Hi! Sorry to jump in 😄  This was removed to avoid a linting warning (see [#1589](https://github.com/nf-core/tools/issues/1589)). It's just a warning, so it won't make linting fail if you add it 👍

Reply:
Great info, thanks!
However, why make in this version all labels black but change a black label to grey just to avoid a warning, why not adjust the warning to include the black label? (The latter seems more logic to me, but maybe this wasnt done to avoid other complications)

Reply:
> Hi! Sorry to jump in 😄 This was removed to avoid a linting warning (see [#1589](https://github.com/nf-core/tools/issues/1589)). It's just a warning, so it won't make linting fail if you add it 👍

Good to know, thanks!

Reply:
> why not adjust the warning to include the black label?

Agree, now that I see that all labels are black, this makes much more sense.